APE14.rst

```diff
@@ -0,0 +1,450 @@
+
+        @property
+        def world_axis_physical_types(self):
+            """
+            Returns an iterable of strings describing the physical type for each
+            world axis. They should be names from the VO UCD1+ controlled
+            Vocabulary (http://www.ivoa.net/documents/latest/UCDlist.html).
+            If no matching UCD type exists, this can instead be "custom:xxx",
+            where xxx is an arbitrary string.  Alternatively, if the physical
+            type is unknown/undefined, an element can be `None`.
+            """
+
+        @property
+        def world_axis_units(self):
+            """
+            Returns an iterable of strings given the units of the world
+            coordinates for each axis. The strings should follow the recommended
+            VOUnit standard (though as noted in the VOUnit specification
+            document, units that do not follow this standard are still allowed,
+            but just not recommended).
+            """
+
+        @property
+        def axis_correlation_matrix(self):
+            """
+            Returns an (n_world, n_pixel) matrix that indicates using booleans
+            whether a given world coordinate depends on a given pixel coordinate.
+            This should default to a matrix where all elements are True in the
+            absence of any further information. For completely independent axes,
+            the diagonal would be True and all other entries False.
+            """
+
+        def pixel_to_world_values(self, *pixel_arrays):
+            """
+            Convert pixel coordinates to world coordinates. This method takes
+            n_pixel scalars or Python array types as input, and pixel
+            coordinates should be zero-based.
+            Returns n_world scalars or arrays in units given by ``world_axis_units``.
+            """
+
+        def world_to_pixel_values(self, *world_arrays):
+            """
+            Convert world coordinates to pixel coordinates. This method takes
+            n_world scalars or arrays as input in units given by ``world_axis_units``.
+            Returns n_pixel scalars or arrays.
+            """
+
+        @property
+        def world_axis_object_components(self):
+            """
+            A list with n_dim_world elements, where each element is a tuple with
+            two items. The first is a name for the world object this world array
+            corresponds to, which *must* match the string names used in
+            ``world_axis_object_metadata``. Note that names might appear twice
+            because two world arrays might correspond to a single world object
+            (e.g. a celestial coordinate might have both “ra” and “dec” arrays,
+            which correspond to a single sky coordinate object). The second
+            element is  either a string keyword argument name or a positional
+            index for the corresponding class from
+            ``world_axis_object_metadata``.
```

Comment:
Two suggestions for making this clearer:
- Use a bullet list for the elements.
- Reference the example below (e.g. "see example below").

Reply:
Fixed